docs/walkthrough/working-with-elasticsearch.md

```diff
@@ -56,3 +56,77 @@ elastic.search(
     size=1000,
 )
 ```
+
+## Monitoring your Elasticsearch cluster
+
+It's helpful to periodically review the Quilt Elasticsearch cluster
+health to ensure performance is optimal and cost-effective.
+
+To review your Elasticsearch cluster:
+
+1. In CloudFormation, select your Quilt stack.
+1. In the "Resources" tab, type `AWS::Elasticsearch::Domain`. There
+will be one result (`Logical ID` value of `Search`).
+1. Click the link in the `Physical ID` column to access the Quilt search cluster.
+1. Select the "Cluster health" tab.
+1. Review the "Summary" section (look for **Green** Status):
+   - If your Status is **Red**, review the [AWS
+   documentation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/handling-errors.html#handling-errors-red-cluster-status)
+   and notify your Quilt account manager.
+   - If your Status is **Yellow**, review the [AWS
+   documentation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/handling-errors.html#handling-errors-yellow-cluster-status)
+   and notify your Quilt account manager.
```

Comment:
Let's not encourage any remedies as that could cause the stack to drift from template. Remedy here is send the info to your Quilt admin. This doc should only be about how to check the health of the cluster, not how to fix it again to avoid drift versus the variant.